src/language-service/src/schemas/ui-lovelace.ts

```diff
@@ -552,6 +553,16 @@ export interface CastEntityConfig {
   hide_if_unavailable?: boolean;
 }
 
+export interface ButtonEntityConfig {
+  type: "button";
+  name: string;
+  icon?: string;
+  action_name?: string;
+  tap_action: ActionConfig;
```

Comment:
Tap action is optional?

```suggestion
  tap_action?: ActionConfig;
```